nearlib/account.js

```diff
@@ -53,7 +53,7 @@ class Account {
             createAccount,
             signature,
         });
-        return await this.nearClient.submitTransaction(signedTransaction);
+        return this.nearClient.submitTransaction(signedTransaction);
```

Comment:
most other code in nearlib does an await for the async functions in nearlib. Why do it differently here?

Reply:
https://eslint.org/docs/rules/no-return-await

Reply:
Apparently we don't run eslint, even though we have the config?

Reply:
@ilblackdragon adding step to gitlab config